src/app/meetups/view-meetups/meetups-view.component.html

```diff
@@ -34,8 +34,10 @@
       <ng-template #memberList>
         <ul>
           <li *ngFor="let mem of members">
-            <span *ngIf="mem === currentUserName; else otherMember">{{mem}} (You)</span>
-            <ng-template #otherMember><span>{{mem}}</span></ng-template>
+            <a routerLink="/manager/users/profile/{{mem}}">
```

Comment:
we should try to not use manager route as even non admin should be looking into profile. manager route is for admin only.